frontend/src/metabase/components/Select.jsx

```diff
@@ -36,6 +36,7 @@ class BrowserSelect extends Component {
     children: PropTypes.array.isRequired,
     onChange: PropTypes.func.isRequired,
     value: PropTypes.any,
+    defaultValue: PropTypes.any,
```

Comment:
Any thoughts on adding this prop to `Select`? An alternative is to just do `<Select value={value || defaultValue}>`.

Reply:
It seems like if we add `defaultValue` to `Select` we should also use that as the placeholder if `value` and `placeholder` are `undefined`.

For now I'd probably just do `<Select value={value !== undefined ? value : defaultValue}>`